src/data_structures/hare_tortoise_algorithm.md

```diff
@@ -0,0 +1,105 @@
+
+---
+tags:
+  - Translated
+e_maxx_link: hare_tortoise_algorithm
+---
+
+# Floyd's Linked List Cycle Finding Algorithm
+
+Given a linked list where the starting point of that linked list is denoted by `head`, and there may or may not be a cycle present. For instance:
+
+![[tortoise_hare_algo.png]]
+
+Here we need to find out the point `C`, i.e the starting point of the cycle.
+
+## Proposed algorithm
+The algorithm is called `Floyd’s Cycle Algorithm or Tortoise n Hare algorithm`.
```

Comment:
Those backticks are used for code snippets. Better would be to write it bold. with `**Floyd's Cycle ...**`. Also it's Tortoise AND Hare, not N.

Reply:
Fixed, changed backticks to bold.